### PR TITLE
Rails 5, here we go.

### PR DESCRIPTION
**Pull request by ricardovaleriano**
# What is it ❓ 

A step in the future.
With this, we can say goodbye to old pieces of stuff on our stack.

## How it works ❓ 

Updates _Rails_ to 5.0 (and also _Ruby_ to 2.3.4), the very meaning of cutting edge. :P

### Should I squash the commits ❓ 

I don't think so because they can be applied in the order that they are right now.
But I can if we decide so. =)

#### Warnings ⚠️ 

To run specs right now shows a bunch of new warnings. A BUNCH.
Part of them inside the gems.
I'm not sure what to do here, should we wait for new versions?
Really don't know.

## Is it any good ❓ 

The specs are passing, but I would like someone more familiar with the running app to do some manual checkings. I can have overlooked something during the changing process.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.2.2
+2.3.4
```
Comment (hosseintoussi) [inline .ruby-version]: Do you think we can move this to `2.4` 💃 ?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -22,7 +22,7 @@ def markdown(text)
       }
       markdown_to_html = Redcarpet::Markdown.new(coderayified, options)
       markdown_to_html.render(text).html_safe
-    rescue StandardError => e
+    rescue StandardError
```
Comment (hosseintoussi) [inline app/helpers/application_helper.rb]: 👏

File: app/views/cards/_likes.html.erb [hunk 3]
```diff
@@ -1,5 +1,8 @@
 <% if current_user %>
-<%= link_to card_like_or_unlike_path(card), method: :patch, remote: true do %>
+  <%= link_to card_like_or_unlike_path(card),
```
Comment (hosseintoussi) [inline app/views/cards/_likes.html.erb]: 👏

File: .ruby-version (hunk 1 above)
Comment (ricardovaleriano) [inline .ruby-version]: For sure! I'm a moron! XD

**Comment by ricardovaleriano** [comment]
@hosseintoussi Don't forget to run this on you environment a click a little bit here, a little bit there, please XD

**Pull request merged**